studio-docs/source/metrics/field-usage.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Measuring field usage in Apollo Studio
+sidebar_title: Field usage
+description: Understand which fields your clients use
+---
+
+In Apollo Studio, your graph's **Fields** page can display usage metrics for each field in your schema. For example, here are metrics for the fields of a `BuildError` type:
+
+<img src="../img/fields-page.jpg" class="screenshot" alt="Fields page in Studio" width="700" />
+
+You can click any field in the table to view details about which clients and operations contribute to its usage.
+
+The table displays some combination of the following metrics for each field, depending on what data is available:
+
+| Metric | Description |
+|--------|-------------|
+| **Field executions** in the last day | How many times your servers have executed the resolver for the field, according to [trace data](/metrics/usage-reporting/) provided by your servers. |
```

Comment:
I am not sure how useful the "trace data" link is. It links to a page that covers all of usage reporting, not just traces. And there's the interesting wrinkle that while you're going to need to be creating traces while executing the operation in order for it to contribute to this field, the data does not have to be *sent to us* in the form of a trace to affect this.  Maybe just leave off the second half of the sentence.